src/less/widgets/bars.less

```diff
@@ -1,15 +1,19 @@
+/* Pushdown Styles */
+.pf-push-down {
+  transition: margin-top .3s;
+  margin-top: 0;
+
+  &.opened {
+    margin-top: 55px;
+  }
+}
+
 .pf-widget-bar {
-  background: @backgroundDefault;
-  color: @colorDefault;
   height: auto;
```

Comment:
bootstrap also does some wacky stuff with what i assume is the form styles:

![sample](http://mrtr.co/3G3Q3w050a2E/Screen%20Shot%202016-04-07%20at%2012.40.39%20PM.png)

can probably address bootstrap stuff in another ticket / pr if it makes more sense.